src/pages/SubTopic.tsx

```diff
@@ -39,7 +40,26 @@ const Topic = () => {
     }
   }, []);
 
-  useResizeObserver(containerRef, resizeObserverOptions, onResize);
+  useEffect(() => {
+    console.log(location.pathname);
+    containerRef.current?.addEventListener('scroll', () => {
+      localStorage.setItem(
+        `${location.pathname}_scrollPosition`,
```

Comment:
let's give it a more meaningful name. 
```suggestion
        `knowledge_hub_${location.pathname}_scrollPosition`,
```

And we should create a variable for this to be reuse in line 55